src/app/components/modules/Settings.jsx

```diff
@@ -157,6 +157,12 @@ class Settings extends React.Component {
         const locale = event.target.value;
         const userPreferences = { ...this.props.user_preferences, locale };
         this.props.setUserPreferences(userPreferences);
+
+        if (locale === 'he') {
```

Comment:
I'm not sure that this piece of code is correctly written. Yeah, it works, but I do use the DOM directly..

Reply:
while your code will definitely cause the attribute to change after a user changes their language to hebrew, the setting won't persist to the next session. i think the right way to do this would be to connect the attribute to [the redux state](https://github.com/steemit/condenser/blob/master/src/app/redux/AppReducer.js#L34).